src/UnitySampleProject/Assets/Samples/web3.unity SDK Web3Auth/2.6.0/Web3Auth Samples/Scripts/Web3AuthLoginProvider.cs

```diff
@@ -78,61 +77,54 @@ protected override async void Initialize()
         base.Initialize();
         providerAndButtonPairs.ForEach(p =>
             p.Button.onClick.AddListener(delegate { LoginWithWeb3Auth(p.Provider); }));
-#if !UNITY_EDITOR && UNITY_WEBGL
-        Web3AuthWebGLConnected += Web3AuthSet;
-        var projectSettings = ProjectConfigUtilities.Load();
-        SetLoginCallback(Web3AuthConnected);
-        //1155 is a decimal number, we need to convert it to an integer
-        InitWeb3Auth(clientId, new HexBigInteger(BigInteger.Parse(projectSettings.ChainId)).HexValue, 
-        projectSettings.Rpc, projectSettings.Network, "", projectSettings.Symbol, "", network.ToString().ToLower());
-#else
+// #if !UNITY_EDITOR && UNITY_WEBGL
+//         Web3AuthWebGLConnected += Web3AuthSet;
+//         var projectSettings = ProjectConfigUtilities.Load();
+//         SetLoginCallback(Web3AuthConnected);
+//         //1155 is a decimal number, we need to convert it to an integer
+//         InitWeb3Auth(clientId, new HexBigInteger(BigInteger.Parse(projectSettings.ChainId)).HexValue, 
+//         projectSettings.Rpc, projectSettings.Network, "", projectSettings.Symbol, "", network.ToString().ToLower());
+// #else
         if (!string.IsNullOrEmpty(KeyStoreManagerUtils.getPreferencesData(KeyStoreManagerUtils.SESSION_ID)))
         {
             rememberMe = true;
-            await TryLogin();
+            await TryConnect();
             Debug.Log("Restoring existing Web3Auth session (Remember Me");
         }
-#endif
+// #endif
     }
-#if !UNITY_EDITOR && UNITY_WEBGL
-    private async void Web3AuthSet(string sessionId)
-    {
-        Web3AuthWebGLConnected -= Web3AuthSet;
-        KeyStoreManagerUtils.savePreferenceData(KeyStoreManagerUtils.SESSION_ID, sessionId);
-        await TryLogin();
-    }
-#endif
-
-#if !UNITY_EDITOR && UNITY_WEBGL
-    [MonoPInvokeCallback(typeof(Action))]
-    private static void Web3AuthConnected(string sessionId)
-    {
-        Web3AuthWebGLConnected?.Invoke(sessionId);
-    }
-#endif
+// #if !UNITY_EDITOR && UNITY_WEBGL
```

Comment:
Can this then be  removed? I did see jslib up, but don't know if you even called methods from there in this file?

Reply:
Makes sense!